docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -0,0 +1,446 @@
+| -----------------------------  |  ----------------------------- |
+| Total Imported events          |                         272572 |
+| Events Imported in this Run    |                         272572 |
+| Ingestion Rate (last 3 mins)   |               14542 events/sec |
+| Ingestion Rate (last 10 mins)  |               14542 events/sec |
+| Time taken in this Run         |                      0.83 mins |
+| Remaining Events               |                        4727427 |
+| Estimated Time to catch up     |                          5m42s |
+| -----------------------------  |  ----------------------------- |
+```
+
+The entire import process is designed to be _restartable_ if yb-voyager terminates when the data import is in progress. If restarted, the data import resumes from its current state.
+
+{{< note title="Note">}}
+The arguments `table-list` and `exclude-table-list` are not supported in live migration.
+For details about the arguments, refer to the [arguments table](../../reference/data-migration/import-data/#arguments).
+{{< /note >}}
+
+{{< tip title="Importing large datasets" >}}
+
+When importing a very large database, run the import data command in a `screen` session, so that the import is not terminated when the terminal session stops.
+
+If the `yb-voyager import data` command terminates before completing the data ingestion, you can re-run it with the same arguments and the command will resume the data import operation.
+
+{{< /tip >}}
+
+#### Import data status
+
+Run the `yb-voyager import data status --export-dir <EXPORT_DIR>` command to get an overall progress of the data import operation.
+
+### Archive changes (Optional)
+
+As the migration continuously exports changes on the source database to the `EXPORT-DIR`, the disk utilization continues to grow indefinitely over time. To limit usage of all the disk space, optionally, you can use the `archive changes` command as follows:
+
+```sh
+yb-voyager archive changes --export-dir <EXPORT-DIR> --move-to <DESTINATION-DIR> --delete
+```
+
+Refer to [archive changes](../../reference/cutover-archive/archive-changes/) for details about the arguments.
+
+### Cutover to the target
+
+During cutover, you switch your application over from the source database to the target YugabyteDB database.
+
+Keep monitoring the metrics displayed for export data and import data processes. After you notice that the import of events is catching up to the exported events, you are ready to perform a cutover. You can use the "Remaining events" metric displayed in the import data process to help you determine the cutover.
+
+Perform the following steps as part of the cutover process:
+
+1. Quiesce your source database, that is stop application writes.
+1. Perform a cutover after the exported events rate ("ingestion rate" in the metrics table) drops to 0 using the following command:
+
+    ```sh
+    yb-voyager cutover initiate --export-dir <EXPORT_DIR> --prepare-for-fall-back
```

Comment:
`yb-voyager cutover initiate --export-dir <EXPORT_DIR> --prepare-for-fall-back true`